tsconfig.tsbuildinfo

```diff
@@ -0,0 +1 @@
+{"root":["./x-element.js"],"version":"5.6.3"}
```

Comment:
Not sure what this is all about. Can we `.gitignore` this?

Reply:
@theengineear You **could** merge the contents into the tsconfig.json. This file is helpful for splitting the project into multiple [project references](https://www.typescriptlang.org/docs/handbook/project-references.html). However, TypeScript also just allows you to [define these references](https://www.typescriptlang.org/tsconfig/#references) in the `tsconfig.json` if you need them anyway. It just supports allowing you using a separate config file, for which this file name [is the default](https://www.typescriptlang.org/tsconfig/#tsBuildInfoFile). I don't know why someone would choose to maintain a separate file from ts-config-- that probably makes more sense from the perspective of very large projects.

Anyway, since x-element is intended to be a single file import, I don't imagine you'll need this unless you're planning to add accessory projects to the repo. I'd just delete it for now. The version property doesn't appear to be an officially supported property for tsconfig, and you're already using the `files` attribute instead of `root`.

Reply:
Thanks for the detailed response! I expected to be the one answering my own question there 😅 

I’ll ignore the file 👌